dask/dataframe/io/io.py

```diff
@@ -1074,5 +1075,30 @@ def from_map(
     return new_dd_object(graph, name, meta, divisions)
 
 
+def to_backend(ddf: _Frame, backend: str | None = None, **kwargs):
```

Comment:
Minor comment: Maybe use `DataFrame | Series | Index` instead of `_Frame` here. This is a user-facing API and most users won't be familiar with `_Frame` but will be with `DataFrame`, `Series`, and `Index`

Reply:
Changing this to `DataFrame | Series | Index` makes `mypy` unhappy, because this function is used by `_Frame.to_backend`. Does it make sense to simply list those more-familiar types in the docstring?